modules/fluid_properties/include/utils/FPNumbers.h

```diff
@@ -0,0 +1,61 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef FPNUMBERS_H
+#define FPNUMBERS_H
+
+#include "libmesh/libmesh_common.h"
+using namespace libMesh;
+
+/**
+ * Compute Reynolds number
+ *
+ * @param volume_fraction  The volume fraction of the phase
+ * @param rho  The density of the phase
+ * @param vel  The velocity of the phase
+ * @param D_h  The hydraulic diameter
+ * @param visc  The viscosity of the phase
+ * @return Reynolds number
+ */
+Real Reynolds(Real volume_fraction, Real rho, Real vel, Real D_h, Real visc);
+
+/**
+ * Compute Prandtl number
+ *
+ * @param cp  Specific heat
+ * @param mu  Dynamic viscosity
+ * @param k  Thermal conductivity
+ * @return Prandtl number
+ */
+Real Prandtl(Real cp, Real mu, Real k);
+
+/**
+ * Compute Grashof number
+ *
+ * @param beta  Thermal expansion coefficient
+ * @param dT  |T_w - T|
+ * @param D_h  Hydraulic diameter
+ * @param rho_l  Density of liquid
+ * @param visc_l  Viscosity of liquid
+ * @param gravity_magnitude  Gravitational acceleration magnitude
+ * @return Grashof number
+ */
+Real Grashof(Real beta, Real dT, Real D_h, Real rho_l, Real visc_l, Real gravity_magnitude);
+
+/**
+ * Compute Laplace number (or coefficient)
+ *
+ * @param surf_tension  Surface tension
+ * @param delta_rho  Difference in density of phases
+ * @param gravity_magnitude  Gravitational acceleration magnitude
+ * @return Laplace number
+ */
+Real Laplace(Real surf_tension, Real delta_rho, Real gravity_magnitude);
```

Comment:
I don't even recognize this Laplace number. The one I know of is
```
La = sigma * rho * L / mu^2
```

Reply:
This is from TRACE, just for the record. I will recode this function...